### PR TITLE
generator: add support for committees

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/community/pull/3298#issuecomment-467217343

`sigs.yaml` includes all our community groups except committees. This PR adds support for committees in the generator.

Note: since committees can have private mailing lists, I have added a new field `private_mailing_list` in the Contacts section. PTAL.

This PR also adds the Steering Committee, the Code of Conduct Committee and the Product Security Committee in `sigs.yaml`. I'll cc them on the PR for approval for their respective sections once the generator changes get lgtm (but will send an email to the steering ML now to get a general ok for the change).

/assign @spiffxp 
generator owner

/sig contributor-experience
/committee steering
/committee conduct
/committee product-security

### CODE REVIEW

File: committee-code-of-conduct/README.md [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
+
+The below teams can be mentioned on issues and PRs in order to get attention from the right people.
+Note that the links to display team membership will only work if you are a member of the org.
+
+| Team Name | Details | Description |
+| --------- |:-------:| ----------- |
+| @kubernetes/code-of-conduct-committee | [link](https://github.com/orgs/kubernetes/teams/code-of-conduct-committee) | General Discussion |
+
+<!-- BEGIN CUSTOM CONTENT -->
+## Terms
```
Comment (nikhita) [inline committee-code-of-conduct/README.md]: I wasn't sure where this information should go, so I put it in the custom content as "Terms". PTAL if this is ok.

File: committee-steering/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* Public Steering Committee Meeting: [Wednesdays at 13:00 PT (Pacific Time)](https://bit.ly/k8s-steering-wd) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting recordings](https://www.youtube.com/watch?v=YAzgJRQxsdc&list=PL69nYSiGNLP1yP1B_nd9-drjoxp0Q14qM).
+
+## Members
+
+* Brian Grant (**[@bgrant0607](https://github.com/bgrant0607)**), Google
+* Brendan Burns (**[@brendanburns](https://github.com/brendanburns)**), Microsoft
+* Derek Carr (**[@derekwaynecarr](https://github.com/derekwaynecarr)**), Red Hat
+* Davanum Srinivas (**[@dims](https://github.com/dims)**), Huawei
+* Joe Beda (**[@jbeda](https://github.com/jbeda)**), VMware
```
Comment (nikhita) [inline committee-steering/README.md]: @jbeda and @timothysc should your affiliation be Heptio or VMware? It says Heptio in the steering repo: https://github.com/kubernetes/steering#members :)

**Comment by nikhita** [comment]
Sent an email to steering ML: https://groups.google.com/a/kubernetes.io/d/topic/steering/iDkt83VedfQ/discussion

**Comment by bgrant0607** [comment]
Cool! Good idea. Thanks a lot. Probably should be called subgroups.yaml at this point, but I think changing the name would be disruptive.

/approve

**Comment by nikhita** [comment]
cc @kubernetes/steering-committee @kubernetes/code-of-conduct-committee @kubernetes/product-security-committee 

I'll take https://github.com/kubernetes/community/pull/3437#pullrequestreview-215066701 as lgtm from steering but will hold for comments.

CoCC and PSC - can you PTAL at the changes and add lgtm if you are ok with it? Thanks! :)

**Comment by nikhita** [comment]
I'll treat @philips's lgtm as approval from PSC (and steering) and @carolynvs's lgtm as approval from CoCC.

Will leave this open until Wednesday and then remove the hold.

**Comment by joelsmith** [comment]
@nikhita, Do we need to go update `OWNERS` and `OWNERS_ALIASES` files that reference `product-security-committee` across all the repos? I see the `product-security-committee` alias instead of `committee-product-security` in these files:

https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/OWNERS
https://github.com/kubernetes/sig-release/blob/master/OWNERS_ALIASES
https://github.com/kubernetes/org/blob/master/OWNERS_ALIASES
https://github.com/kubernetes/org/blob/master/config/kubernetes/org.yaml
https://github.com/kubernetes/security/blob/master/OWNERS
https://github.com/kubernetes/security/blob/master/OWNERS_ALIASES

I'm happy to make PRs to update those 3 repos with the new name if that's the right thing to do.

**Comment by nikhita** [comment]
>  Do we need to go update OWNERS and OWNERS_ALIASES files that reference product-security-committee across all the repos? 

TLDR: No, that's not needed. :)

I updated the alias name here because the current format works well with the go template in the generator  (which generates the `OWNER_ALIASES` file from `sigs.yaml`) i.e. the only way to get the complete committee's name using the fields in sigs.yaml is to use `{{.Dir}}` and that uses the word `committee` as a prefix.
